### ISSUE TITLE
[BUG] MCC crashes when player uses Elytra and Rockets

### ISSUE THREAD

**Issue by EpicnessTwo** [MCCTeam/Minecraft-Console-Client#2425]
### Prerequisites

- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one
- [X] I have set `debugmessages=true` in config to diagnose my issue
- [X] I have redacted session tokens and passwords before attaching screenshots

### Minecraft Version

1.19.3

### Console Client Version

Latest

### Expected Behavior

The bot doesn't crash

### Actual Behavior

The bot crashes

### Steps to Reproduce the bug

1. Enable EntityHandling inside the config
2. On another account, use an elytra and a rocket to fly within the loaded area for the bot
3. The bot will crash

### Anythings that could help diagnosing the bug

When flying around the bot, everything works just fine. If you set off rocket, it still works fine. It's only when you combine the two actions and fly with a rocket, the bot will crash with the following message. I've enabled debugging and nothing extra is returned.

This has been replicated on 2 seperate MCC bots at the exact same time. One running Windows and the other running Linux to rule out issues with the binary for that OS. Disabling EntityHandling will stop the bot from crashing but then prevents actions such as the AutoFish module.

```
> Unhandled exception. System.InvalidOperationException: Queue empty.
   at System.Collections.Generic.Queue`1.ThrowForEmptyQueue()
   at System.Collections.Generic.Queue`1.Dequeue()
   at MinecraftClient.Protocol.Handlers.DataTypes.ReadNextMetadata(Queue`1 cache, ItemPalette itemPalette)
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket(Int32 packetID, Queue`1 packetData)
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.Updater(Object o)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
Aborted (core dumped)
```

### Device

Other

### Operating System

Linux

### Server Address (If applicable)

play.epiccraft.network

**Comment by milutinke** [comment]
Metadata handling was not updated yet for 1.19.3 if I remember correctly.
Thanks for the report.

**Comment by milutinke** [comment]
Fixed in #2447 
Will be available as a pre-release soon.
